sonarts-core/docs/rules/no-empty-statements.md

```diff
@@ -0,0 +1,22 @@
+# no-empty-statements
+
+Empty statements, i.e. `;`, are usually introduced by mistake, for example because:
+
+<ul>
+  <li> It was meant to be replaced by an actual statement, but this was forgotten. </li>
+  <li> There was a typo which lead the semicolon to be doubled, i.e. `;;`. </li>
+</ul>
+## See
```

Comment:
@saberduck for some reason this title is not displayed correctly. May be no empty line before matters